projects/README.md

```diff
@@ -0,0 +1,128 @@
+# 🗂️ Final Challenge
+
+Now that you have learned about the different aspects of building with efficiency in mind, we propose you to pick a final challenge where you can apply all the knowledge you have gained during the course.
+
+In this section, we will present you four different projects with a central theme. Guess it? Efficiency, of course, in a real-world case.
+
+Before you start, check the following pointers:
+
+- The challenges are **unsupervised**. So, we are not providing you with an official solution. You are free to experiment, explore and be as creative as you want. The is no right or wrong way to complete the challenge.
+
+- We recommend you to **select a challenge you are genuinely interested in**. If you are unsure where to begin, check the provided tools and resources. You can start with the easiest level first and then develop the more advanced features.
+
+- Reach out to the **[Discord community](https://discord.com/invite/Tun8YgzxZ9)**. If you are stuck, we will be happy to discuss your ideas and help you get started. If you have completed the challenge, we would be excited to see and share your project.
+
```

Comment:
perhaps it would be cool to encourage to publish open source and share on socials, where we will reshare?